app/models/assessment_details_review.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
```

Comment:
A form object! I will take kate on her word as I don't have the skill to review this

Reply:
I thought I'd leave a bit of an explanation here for posterity!

This is a form object. See [this blog post](https://thoughtbot.com/blog/activemodel-form-objects) for a good introduction to form objects.

The basic idea is that by `ActiveModel::Model` we can make a plain Ruby object active like a Rails Active Model.